corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserEditor.java

```diff
@@ -93,6 +95,10 @@ public CorfuStoreBrowserEditor(CorfuRuntime runtime, String diskPath) {
         runtime.getSerializers().registerSerializer(dynamicProtobufSerializer);
     }
 
+    public void setSnapshot(Token snapshot) {
```

Comment:
I think this change should come along with another utility that gives a list of available timestamps (otherwise it will be hard to be used by a common user). Something of the type --table myTable --getSnapshots